code/modules/hydroponics/grown/nettle.dm

```diff
@@ -74,32 +77,30 @@
 	. = ..()
 	if(!proximity)
 		return
-	if(force > 0)
-		force -= rand(1, (force / 3) + 1) // When you whack someone with it, leaves fall off
-	else
-		to_chat(usr, "<span class='warning'>All the leaves have fallen off the nettle from violent whacking.</span>")
-		qdel(src)
+	if(force > minforce)
+		force = max(force - rand(0,2),minforce) // When you whack someone with it, leaves fall off
+		if(force == minforce)
+			icon_state = bareicon
 
 /obj/item/food/grown/nettle/basic
 	seed = /obj/item/seeds/nettle
 
-/obj/item/food/grown/nettle/basic/Initialize(mapload, obj/item/seeds/new_seed)
+/obj/item/food/grown/nettle/Initialize(mapload, obj/item/seeds/new_seed)
 	. = ..()
-	force = round((5 + seed.potency / 5), 1)
+	minforce = round(minforce * (1 + seed.potency / 200))
+	force = round((minforce + seed.potency / 10), 1)
 
 /obj/item/food/grown/nettle/death
 	seed = /obj/item/seeds/nettle/death
 	name = "deathnettle"
 	desc = "The <span class='danger'>glowing</span> nettle incites <span class='boldannounce'>rage</span> in you just from looking at it!"
 	icon_state = "deathnettle"
-	force = 30
+	bareicon = "deathnettle_bare"
+	force = 20
```

Comment:
won't a potency 100 deathnettle start with a force of only 18, not 20, under your new system?